tests/test_storage.py

```diff
@@ -0,0 +1,33 @@
+from memgpt.connectors.storage import StorageConnector, Passage
+from memgpt.connectors.db import PostgresStorageConnector
+from memgpt.embeddings import embedding_model
+from memgpt.config import MemGPTConfig
+
+import argparse
+
+
+def test_postgres():
+    config = MemGPTConfig.load()
+    embed_model = embedding_model()
+
+    passage = ["This is a test passage", "This is another test passage", "Cinderella wept"]
+
+    db = PostgresStorageConnector(uri=config.archival_storage_uri, table_name="test2")
```

Comment:
This throws an error:

```
ERROR tests/test_storage.py - TypeError: PostgresStorageConnector.__init__() got an unexpected keyword argument 'uri'
```

Because of the signature here:

```

class PostgresStorageConnector(StorageConnector):
    """Storage via Postgres"""

    # TODO: this should probably eventually be moved into a parent DB class

    def __init__(self, name: Optional[str] = None, agent_config: Optional[AgentConfig] = None):
        config = MemGPTConfig.load()
```